src/systems/renderer.js

```diff
@@ -21,7 +21,8 @@ module.exports.System = registerSystem('renderer', {
     colorManagement: {default: false},
     gammaOutput: {default: false},
     alpha: {default: true},
-    foveationLevel: {default: 0}
+    foveationLevel: {default: 0},
+    useWebGL1: {default: false}
```

Comment:
let's call this `wegbl2` (default false).

Reply:
So you want users to use AFrame-1.1.0 with webgl1 by default?